src/main/java/arona/Ui.java

```diff
@@ -0,0 +1,68 @@
+package arona;
+
+import java.util.ArrayList;
+import java.util.Scanner;
+
+public class Ui {
+    private String logo =        "    _____ \n" +
+            /* */                "   /  _  \\_______   ____   _____ _____ \n" +
+            /*   */              "  /  /_\\  \\_  __  \\/  _ \\ /     \\___  \\ \n" +
+            /*   */              " /    |    \\  | \\_ ( <_> )   |   \\/ __ \\_ \n" +
+            /* */                " \\____|__  /__|    \\____/|___|_  (____  / \n" +
+            /* */                "         \\/                    \\/     \\/ ";
+    private String greetings = "こんにちは先生、私はアロナです. \n"
```

Comment:
I like your use of Japanese, but perhaps it could be confusing for some users to read?